src/plugins/home/public/application/components/home.js

```diff
@@ -155,6 +155,7 @@ export class Home extends Component {
           showDevToolsLink
           showManagementLink
           title={<FormattedMessage id="home.header.title" defaultMessage="Home" />}
+          branding={getServices().injectedMetadata.getBranding()}
```

Comment:
is this related to this PR?

Reply:
Yes. Actually the overviewHeader component is used twice in the repository. One for overview page header, another one for home page header. Since we add branding as one of the prop of overviewHeader component, we need to pass the branding here in home.js as well. 

It has no effect on home component and It will not render any logo on the home page header because of the logic within overviewHeader component.